.circleci/config.yml

```diff
@@ -40,6 +40,13 @@ aliases:
         ignore:
           - master
 
+  ignore_docs: &ignore_docs
+    filters:
+      branches:
+        ignore:
+          - /docs(\/)?.*/
```

Comment:
NIT: is `(\/)?` needed if it's followed by `.*` anyway?

Reply:
What about `docs.+`? It seems unlikely we need to account for raw `docs`, `blog`, etc. anyways.